apps/web/src/views/PositionManagers/components/AprButton.tsx

```diff
@@ -25,6 +25,9 @@ const AprText = styled(Text)`
   cursor: pointer;
 `
 
+const lpTokenDecimals = 8 // the vault ABI from 3 party is difference, so we need to hardcode this, just for user can input the decimal points
```

Comment:
Do you think should we add this in config ? 
In case each of them needs to show different decimals.
Not sure have this case or not, if not just ignore my msg.

Reply:
I read it from the contract now, I checked with PM that we can always get it from the vault address